src/sidebar/services/load-annotations.js

```diff
@@ -39,6 +41,7 @@ export default function loadAnnotationsService(
   function searchAndLoad(uris, groupId) {
     searchClient = new SearchClient(api.search, {
       incremental: true,
+      separateReplies: !features.flagEnabled('client_do_not_separate_replies'),
```

Comment:
As mentioned, this logic is in the service because `SearchClient` can't check features (AFAIK).

Reply:
You're right - `SearchClient` is just a standalone utility class that has no access to the `features` service (or any other service for that matter).